src/components/importwms/ImportWmsDirective.js

```diff
@@ -285,39 +297,51 @@ goog.require('ga_urlutils_service');
           39.37 * 72;
     };
 
+    // Adjust layer extent center according to scale
+    var adjustLayerExtentCenter =
+    function(view, extent, mapSize, maxScaleDenominator) {
```

Comment:
put the function on previous line and use 4 spaces indentationon the next line like this:  
```
var adjustLayerExtentCenter = function(view, extent, mapSize,
    maxScaleDenominator) {
```